.circleci/config.yml

```
@@ -54,7 +54,7 @@ jobs:
             yarn test
       - run:
           name: Running tests
-          command: yarn test --coverage
+          command: yarn test --coverage --testPathPattern='packages/platform-sdk/test/(?!integration-tests).*'
```

This change should be reverted to what it was.

```suggestion
          command: yarn test --coverage
```